dist/utilities/border/_border.scss

```diff
@@ -6,12 +6,12 @@
 
 @mixin border(
     $important: false,
-    $border: if(variable-exists(border), $border, null)
```

Comment:
So it allows us to create new utility classes for different border colours eg. 

```
.u-border-light {
    @include border(true, $neutral-15);
}
```

if `$border-color` exist and we can use:

```
.u-border {
    @include border(true);
}
```